examples/js/controls/PointerLockControls.js

```diff
@@ -10,14 +10,7 @@ THREE.PointerLockControls = function ( camera, domElement ) {
 	this.domElement = domElement || document.body;
 	this.isLocked = false;
 
-	camera.rotation.set( 0, 0, 0 );
-
-	var pitchObject = new THREE.Object3D();
-	pitchObject.add( camera );
-
-	var yawObject = new THREE.Object3D();
-	yawObject.position.y = 10;
-	yawObject.add( pitchObject );
+	camera.rotation.reorder( 'YXZ' );
```

Comment:
Could this have a side effect? What if the user disconnects the controls at some point and then changes the rotation manually? The result will be unexpected if the euler order is not the default. 

Could also be problematic if `PointerLockControls` is used with other controls.

Reply:
>Could this have a side effect? What if the user disconnects the controls at some point and then changes the rotation manually?

You can change it back upon disconnect, then.

>Could also be problematic if PointerLockControls is used with other controls, right?

Can you please provide an example?

Reply:
I've thought of a custom control class that also changes the rotation via euler angles. In this case, you have to be aware that `PointerLockControls` changes the euler order and act accordingly.

Reply:
You are referring to hypotheticals.

Reply:
I just think it's a bad design decision to change the euler order in controls.

Reply:
That is an opinion and a preference.  That is fine. 

But to file objections without specifics that the OP can respond to is unfair to the OP -- me. It also has the effect of blocking a PR from ever being merged.

And as I said, the order can be restored upon disconnect -- or at the end of every update if you insist.